unit_tests/sources/declarative/transformations/test_flatten_fields.py

```diff
@@ -10,11 +10,12 @@
 
 
 @pytest.mark.parametrize(
-    "input_record, expected_output",
+    "flatten_lists, input_record, expected_output",
     [
-        ({"FirstName": "John", "LastName": "Doe"}, {"FirstName": "John", "LastName": "Doe"}),
-        ({"123Number": 123, "456Another123": 456}, {"123Number": 123, "456Another123": 456}),
+        (True, {"FirstName": "John", "LastName": "Doe"}, {"FirstName": "John", "LastName": "Doe"}),
```

Comment:
I'm not really attached to this but I'm wondering what is your opinion on this. It is becoming harder and harder when we have a lot of cases to relate to which parameter is what. Do you think it would be overkill to have `_FLATTEN_LISTS = True` and `_DO_NOT_FLATTEN_LISTS = False` and use them here for extra clarity? Or who it be better to have ids for each test cases? Or should we do both?

Reply:
good point, added ids and constants for flatten_fields